Todo.Web/Client/TodoClient.cs

```diff
@@ -11,15 +11,23 @@ public TodoClient(HttpClient client)
         _client = client;
     }
 
-    public async Task<bool> AddTodoAsync(string? title)
+    public async Task<(bool IsSuccessStatusCode, TodoItem? TodoItem)> AddTodoAsync(string? title)
```

Comment:
Can we instead just return a nullable object:
 null - failed
non null - works

Reply:
Sure thing, done: https://github.com/davidfowl/TodoApi/pull/38/commits/229f5ee30c8a73ef180fe755dcc016a4f8501f87